src/data-publication/constants/snapshot-dataset.js

```diff
@@ -6,9 +6,34 @@ const ARID2017_XREF = {
 }
 
 export const SNAPSHOT_DATASET = {
+  2021: {
+    snapshot_date: 'May 4, 2022',
+    datasets: [
+      {
+        csv: 'https://s3.amazonaws.com/cfpb-hmda-public/prod/snapshot-data/2020/2020_public_lar_csv.zip',
```

Comment:
Update to 2021

Reply:
Addressed in follow-up PR for adding 2021 data: https://github.com/cfpb/hmda-frontend/pull/1436/files#diff-3a33bfdc7449537e15122a081cfb9b8249a48e51f3bc5f96019895c644a8c39dR14